src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -17,13 +17,16 @@ import {
   POST_TYPE_POLL,
   POST_TYPE_STANDARD
 } from '../../utils/constants';
-import {Footer, Gap, PreviewComment} from '../../components';
+import {Footer, PreviewComment} from '../../components';
 import {getCommentLength, getCountCommentWithChild} from '../../utils/getstream';
 import {showScoreAlertDialog} from '../../utils/Utils';
-import {normalizeFontSizeByWidth} from '../../utils/fonts';
+import {normalize, normalizeFontSizeByWidth} from '../../utils/fonts';
 import {COLORS} from '../../utils/theme';
-import WriteComment from '../../components/Comments/WriteComment';
 import useWriteComment from '../../components/Comments/hooks/useWriteComment';
+import TopicsChip from '../../components/TopicsChip/TopicsChip';
+import useCalculationContent from '../FeedScreen/hooks/useCalculationContent';
+import BlurredLayer from '../FeedScreen/elements/BlurredLayer';
+import AddCommentPreview from '../FeedScreen/elements/AddCommentPreview';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `Gap` and `WriteComment` have been removed and replaced with `TopicsChip`, `useCalculationContent`, `BlurredLayer`, and `AddCommentPreview`. Ensure that these changes do not affect other parts of the code where `Gap` and `WriteComment` might be used.
```diff
- import {Footer, Gap, PreviewComment} from '../../components';
- import WriteComment from '../../components/Comments/WriteComment';
+ import {Footer, PreviewComment} from '../../components';
+ import TopicsChip from '../../components/TopicsChip/TopicsChip';
+ import useCalculationContent from '../FeedScreen/hooks/useCalculationContent';
+ import BlurredLayer from '../FeedScreen/elements/BlurredLayer';
+ import AddCommentPreview from '../FeedScreen/elements/AddCommentPreview';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->